### PR TITLE
Fix ParallelQuery assumptions on errors

### LINKED ISSUE #338: ParallelQuery error when some but not all requests fail
```
2023-11-03 09:29:50,390 : ERROR : aperturedb.ParallelQuery : 139786824185408 : 242 : 'str' object has no attribute 'items'
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/aperturedb/ParallelQuery.py", line 240, in worker
    self.do_batch(db, generator[batch_start:batch_end])
  File "/usr/local/lib/python3.10/dist-packages/aperturedb/ParallelQuery.py", line 210, in do_batch
    [v['status'] == 0 for i in r for k, v in i.items()])
  File "/usr/local/lib/python3.10/dist-packages/aperturedb/ParallelQuery.py", line 210, in <listcomp>
    [v['status'] == 0 for i in r for k, v in i.items()])
AttributeError: 'str' object has no attribute 'items'

```

### PR DESCRIPTION
**Pull request by drewaogle**
Fixes #338

### CODE REVIEW

**Review by gsaluja9 — APPROVED** [review]
nice. Love the bad command (or file name :) haha )

**Review by gsaluja9 — APPROVED** [review]
Looks good. It'd be good to have this ci.sh at a common place as it might be replicating code in different repos.

File: aperturedb/ParallelLoader.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def print_stats(self):
         else:
             mean = np.mean(times)
             std  = np.std(times)
-            tp = 1 / mean * self.numthreads
+            tp = 0 if mean == 0 else 1 / mean * self.numthreads
```
Comment (gsaluja9) [inline aperturedb/ParallelLoader.py:109]: does this address NaN errors on the [stats](https://github.com/aperture-data/aperturedb-python/issues/244) ?

Comment (drewaogle) [reply aperturedb/ParallelLoader.py:109]: no, that is due to the {i_tp if self.error_counter == 0 else 'NaN'} .. but with the testing I'm adding to check for errors loading images with ImageDataCSV, we can resolve that too.

**Pull request merged**